src/java/org/apache/cassandra/utils/JMXServerUtils.java

```diff
@@ -224,15 +229,15 @@ private static Map<String, Object> configureJmxSocketFactories(InetAddress serve
             String protocolList = COM_SUN_MANAGEMENT_JMXREMOTE_SSL_ENABLED_PROTOCOLS.getString();
             if (protocolList != null)
             {
-                System.setProperty("javax.rmi.ssl.client.enabledProtocols", protocolList);
+                JAVAX_RMI_SSL_CLIENT_ENABLEDPROTOCOLS.setString(protocolList);
```

Comment:
these constants should have nicely readable, whatever their key is, put spaces there between words.

Reply:
Fixed.